routes.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');// pass passport for configuration
+const router = express.Router();
+var User = require('./model/usermodel');
+
+router.get('/alluser',(req,res,next)=>{
+    User.find(function(err,user){
+        res.json(user);
+    });
+  });
+
+router.get('/deleteall',(req,res,next)=>{
+    User.remove({},(err,user)=>{
```

Comment:
Dont make this a get route. make it a POST/DELETE route with the same endpoint of /user as above

Reply:
ok i made it to clear the database as undefined data were stored by mistake